helix-core/src/test/java/org/apache/helix/task/TestGetLastScheduledTaskExecInfo.java

```diff
@@ -29,10 +29,14 @@
 import org.testng.Assert;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
+import org.apache.helix.TestHelper;
 
 public class TestGetLastScheduledTaskExecInfo extends TaskTestBase {
   private final static String TASK_START_TIME_KEY = "START_TIME";
   private final static long INVALID_TIMESTAMP = -1L;
+  private static int SHORT_EXECUTION_TIME = 10;
+  private static int LONG_EXECUTION_TIME = 99999999;
+  private static final int DELETE_DELAY = 30 * 1000;
```

Comment:
nit: make them longs with L appended at the end :)

Reply:
Fixed. Thanks.